src/modules/peek/Peek.UI/MainWindow.xaml

```diff
@@ -37,6 +37,7 @@
         <fp:FilePreview
             Grid.Row="1"
             File="{x:Bind ViewModel.FolderItemsQuery.CurrentFile, Mode=OneWay}"
+            ScalingFactor="{x:Bind ViewModel.ScalingFactor, Mode=OneWay}"
```

Comment:
On a similar note to my previous comment, I'm not sure if it is necessary to bubble up to scale factor to pass it down when it is only needed for the <Image> control that the IBitmapPreviewer uses.

Reply:
I agree, that doesn't seem to be clean, because scaling is needed only for BitmapPreviewer. I'm not sure that there is a better way for that though. Please, correct me if I'm wrong.   
`GetMonitorScale` depends on the window, so anyway some info should be passed from this level.